packages/kit/src/core/make_package/essential_files.js

```diff
@@ -0,0 +1 @@
+export default ['README', 'LICENSE', 'CHANGELOG', '.gitignore', '.npmignore'];
```

Comment:
Not 100% sure if we need an extra file for this

Reply:
Me too, I explained a bit as to why on the second paragraph of https://github.com/sveltejs/kit/pull/1747/#issuecomment-867452394. Hoping to get some feedback first and if it's just these files for now, I can merge it back

Reply:
@ignatiusmb did you link to the right PR / issue in your response here? It links to this PR itself and I don't really see an explanation

Reply:
I combined the files for now and will go ahead and merge this. We can separate it back out when there's more to put there